vector-im/element-x-android
# Hide sender info in direct rooms, and add a timeline preview for DM

## Type of change


## Content


Do not render avatar and display name of the other member in the timeline of a direct room.

## Motivation and context


Parity with iOS
Closes #1979

## Screenshots / GIFs



## Tests



- Open a DM and observe that the avatar and display names are not displayed anymore above the messages.
- Open a regular room and check that the avatars are still there

## Tested devices

- OS version(s):

## Checklist
